### DISCUSSION TITLE
Capgo bundle download/install on new app install on phone

### DISCUSSION THREAD

**Discussion by kinggolf** [riderx/capacitor-updater#232] [General]
Thank you for the wonderful and well documented @Capgo/capacitor-updater plugin. Let's say I have version 2.0.0 on the App & Play Stores, but I have a new bundle version available, 2.0.1 or higher. For a brand new user who is doing an initial download of the app from either App or Play Store, is it possible for this initial install to download the updated bundle version and install immediately?
It seems what happens is the user is presented w/ version 2.0.0 initially, but the update will be already downloaded and on first app background the new version 2.0.1 will be installed.

**Comment by riderx** [comment]
Hello @kinggolf right now the behaviour is the last one, next time user close the app the update will happen.
It's mainly because the process is async.
I have check what i can do for this and i didn't found any good solution for now.

**Reply by kinggolf** [reply]
Thanks. I think the current behavior is completely acceptable. I would consider this more nice-to-have behavior.
